aws_lambda_fsm/fsm.py

```diff
@@ -746,7 +747,7 @@ def dispatch(self, event, obj):
                 # understand fence tokens.
 
                 # make the fence token available
-                if isinstance(fence_token, (int, long)):
+                if isinstance(fence_token, (int, int)):
```

Comment:
`long` is a different type in py2. does `isinstance(fence_token, (types.IntType, type.LongType))` work in py2 and py3?